static/app/views/explore/metrics/metricsFlags.tsx

```diff
@@ -0,0 +1,13 @@
+// Only add feature flag checks to this file to stop inadverent imports.
+
+import type {Organization} from 'sentry/types/organization';
+
+const canUseMetricsUI = (organization: Organization) => {
+  return organization.features.includes('tracemetrics-enabled');
+};
+
```

Comment:
Is there a reason these aren't hooks so we don't need to pass through organization?

Reply:
leaves only one file for flag cleanup, hopefully